lib/fshash/bucket_memory.go

```diff
@@ -0,0 +1,51 @@
+package fshash
+
+import (
+	"sort"
+	"strings"
+
+	"polydawn.net/repeatr/lib/treewalk"
+)
+
+var _ Bucket = &MemoryBucket{}
+
+type MemoryBucket struct {
```

Comment:
Neither of us sound like we have strong opinions on this; I'll leave it at your discretion.